rmb-sdk-go/peer/connection.go

```diff
@@ -34,6 +34,21 @@ type send struct {
 	err  chan error
 }
 
+func (s *send) reply(ctx context.Context, err error) error {
+	defer func() {
+		if r := recover(); r != nil {
+			log.Debug().Err(r.(error)).Send()
```

Comment:
this will panic again. there is absolutely no grantee that the returned "r" is of type error because i can do `panic("hello world")` which will return a string in recover() in this case

Reply:
just format the object as a string instead